src/scmdata/groupby.py

```diff
@@ -111,35 +123,34 @@ def apply(
         Parameters
         ----------
         func
-            Callable to apply to each timeseries.
+            Callable to apply to each group.
 
-        ``*args``
+        *args
             Positional arguments passed to `func`.
 
-        ``**kwargs``
-            Used to call `func(ar, **kwargs)` for each array `ar`.
+        **kwargs
+            Keyword arguments passed to `func`.
 
         Returns
         -------
-        applied : :class:`ScmRun <scmdata.run.ScmRun>`
-            The result of splitting, applying and combining this array.
+        The result of applying and combining.
```

Comment:
Wasn't there a rule to indent the returns section one level deeper because that looks better in the generated docs?

Reply:
Nice pick up, thanks

Reply:
Fixed now